postgresqlPackages.plv8: 2.3.12 -> 2.3.13

marsam opened the pull request:
<!-- Nixpkgs has a lot of new incoming Pull Requests, but not enough people to review this constant stream. Even if you aren't a committer, we would appreciate reviews of other PRs, especially simple ones like package updates. Just testing the relevant package/service and leaving a comment saying what you tested, how you tested it and whether it worked would be great. List of open PRs: <https://github.com/NixOS/nixpkgs/pulls>, for more about reviewing contributions: <https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#sec-reviewing-contributions>. Reviewing isn't mandatory, but it would help out a lot and reduce the average time-to-merge for all of us. Thanks a lot if you do! -->
###### Motivation for this change

Changes: https://github.com/plv8/plv8/compare/v2.3.12...v2.3.13

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

marsam committed:
postgresqlPackages.plv8: 2.3.12 -> 2.3.13

Changes: https://github.com/plv8/plv8/compare/v2.3.12...v2.3.13

mmahut commented:
Fails on ```aarch64```.

```building '/nix/store/xmx42427h48y4jhvlvi9m33p6haqx06k-v8-6.4.388.40.drv'...
unpacking sources
unpacking source archive /nix/store/1g1akgzq8jwzsdbhy108vpyvv99g567w-source
source root is source
patching sources
patching script interpreter paths in tools/dev/v8gen.py
tools/dev/v8gen.py: interpreter directive changed from "/usr/bin/env python" to "/nix/store/xml30xvd7xbv98v6ciivd3yi71frh2cf-python-2.7.16/bin/python"
updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./third_party/icu/source/config.sub
cp: cannot remove './third_party/icu/source/config.sub': Permission denied
builder for '/nix/store/xmx42427h48y4jhvlvi9m33p6haqx06k-v8-6.4.388.40.drv' failed with exit code 1
cannot build derivation '/nix/store/jk8gf1aq87ir6yb1c7bxllvhmbasd5ml-plv8-2.3.13.drv': 1 dependencies couldn't be built
error: build of '/nix/store/jk8gf1aq87ir6yb1c7bxllvhmbasd5ml-plv8-2.3.13.drv' failed```

marsam committed:
postgresqlPackages.plv8: narrow platforms to x86_64-linux

plv8 uses a custom version v8 version, which is reported to only build
on x86_64-linux. The next major version plv8 should drop the custom v8.

marsam committed:
postgresqlPackages.plv8: add marsam as maintainer

marsam commented:
@mmahut thanks for testing, I've narrowed plv8's platforms to x86_64-linux

mmahut commented:
@GrahamcOfBorg build postgresqlPackages.plv8
